docs/usage/instance_set.md

```diff
@@ -0,0 +1,107 @@
+# Maintaining instance replicas using instances-sets
+
+## Setting up your usage environment
+
+* Follow the [steps described here](prerequiste.md)
+
+## Important :warning: 
+- Make sure that the `kubernetes/instance-set.yaml` points to the same class name as the `kubernetes/aws-instance-class.yaml`.
+- Similarily `kubernetes/aws-instance-class.yaml` secret name and namespace should be same as that mentioned in `kubernetes/secret.yaml`
+
+## Creating instance-set
+
+- Modify `kubernetes/instance-set.yaml` as per your requirement. You can modify the number of replicas to your desired number of instances. Then, create an instance-set as shown below
```

Comment:
s/your desired/the desired/
s/create an instance-set as shown below/create an instance-set/